src/standalone/Snow/Snow.jl

```diff
@@ -81,9 +81,4 @@ end
 
 include("./snow_parameterizations.jl")
 
-include("./neural_snow/DataTools.jl")
```

Comment:
in the future, if we want to make a SnowModel struct that uses neural snow, do we just need to load the extensions in the script/driver like you do in the docs examples? What if there are situations where we need the types defined in ModelTools.jl inside ClimaLand/src?

Reply:
No, in the future, we will add to ClimaLand the functions that are relevant to ClimaLand and add their implementation in the extension. Suppose the function you care about is `compute_swe`, we will have `compute_swe` in ClimaLand, and it will be "activated" once add the required modules are loaded.

But I think that we don't need any of the functions defined in ModelTools and DataTools directly in ClimaLand, so I have to fetch them directly from the extension.